staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time.go

```diff
@@ -147,8 +147,12 @@ func (t Time) MarshalJSON() ([]byte, error) {
 		// Encode unset/nil objects as JSON's "null".
 		return []byte("null"), nil
 	}
-
-	return json.Marshal(t.UTC().Format(time.RFC3339))
+	buf := make([]byte, 0, len(time.RFC3339)+2)
+	buf = append(buf, '"')
+	// time cannot contain non escapable JSON characters
+	buf = t.AppendFormat(buf, time.RFC3339)
```

Comment:
does this still UTC normalize?

Reply:
No, I was checking to see if the tests failed (to verify we had some) and forgot to mention that or put hold.  My bad.

/hold

until I verify the tests and add that back